rpcserver.go

```diff
@@ -552,15 +552,13 @@ func (r *rpcServer) MintAsset(ctx context.Context,
 		// If a custom decimal display was requested, add that to the
 		// metadata and re-validate it.
 		if metaType == proof.MetaJson && req.Asset.DecimalDisplay != 0 {
```

Comment:
I think this condition now needs to change and not require the `metaType` to be JSON?

Reply:
Good point! Fixed this and a couple of other occurrences of similar checks.